FWCore/Services/test/test_catch2_SiteLocalConfigService.cc

```diff
@@ -27,7 +27,12 @@ TEST_CASE("Test SiteLocalConfigService", "[sitelocalconfig]") {
 
     CHECK(slc.trivialDataCatalogs()[0] == "trivialcatalog_file:/dummy/storage.xml?protocol=dcap");
     edm::CatalogAttributes tmp("DUMMY", "DUMMY_SUB_SITE", "DUMMY_CROSS_SITE", "CMSXrootdFederation", "XRootD");
-    CHECK(slc.dataCatalogs()[0] == tmp);
+    std::cout << "\n " << slc.dataCatalogs()[0].subSite << " " << tmp.subSite;
```

Comment:
When all tests succeed, could you remove this line?

Reply:
Done